components/raftstore-v2/src/batch/store.rs

```diff
@@ -639,9 +644,22 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
                             return;
                         }
                         warn!(logger, "flushing oldest cf of regions {regions:?}");
+                        let flush_count = EK::get_accumulated_flush_count(None).unwrap();
```

Comment:
> "EK::get_accumulated_flush_count(None)" 

This is another reason to separate the API into cf specific and all cf versions.  It's more intuitive than using a None to indicate all cfs.